dataframes/02-groupby.ipynb

```diff
@@ -0,0 +1,226 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DataFrames: Groupby\n",
+    "\n",
+    "This notebook uses the Pandas groupby-aggregate and groupby-apply on scalable Dask dataframes.  It will discuss both common use and best practices."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Start Dask Client for Dashboard\n",
+    "\n",
+    "Starting the Dask Client is optional.  It will provide a dashboard which \n",
+    "is useful to gain insight on the computation.  \n",
+    "\n",
+    "The link to the dashboard will become visible when you create the client below.  We recommend having it open on one side of your screen while using your notebook on the other side.  This can take some effort to arrange your windows, but seeing them both at the same is very useful when learning."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask.distributed import Client\n",
+    "client = Client(n_workers=1, threads_per_worker=4, processes=False, memory_limit='2GB')\n",
+    "client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Artifical dataset\n",
+    "\n",
+    "We create an artificial timeseries dataset to help us work with groupby operations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import dask\n",
+    "df = dask.datasets.timeseries()\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This dataset is small enough to fit in memory, so we persist it now.\n",
+    "\n",
+    "You would skip this step if your dataset becomes too large to fit into memory."
```

Comment:
I get the feeling these are meant as introductory notebooks. Should we clarify which "memory" may be too small at this point? I could see confusion on whether this is the memory of the client or the cluster.

Reply:
But, if this is introductory, that may be too confusing at this point.

Reply:
I've added the word "cluster's" before memory here to disambiguate